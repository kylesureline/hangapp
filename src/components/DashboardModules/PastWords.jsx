import React from 'react';
import PastWordsList from './PastWordsList.jsx';

export const PastWords = () => (
  <div>
    <h3 className="module__title">Last 10 Words</h3>
    <PastWordsList />
  </div>
);

export default PastWords;
