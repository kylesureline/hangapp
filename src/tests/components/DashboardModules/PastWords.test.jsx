import React from 'react';
import { shallow } from 'enzyme';
import { PastWords } from '../../../components/DashboardModules/PastWords.jsx';

test('should render PastWords correctly', () => {
  const wrapper = shallow(<PastWords />);
  expect(wrapper).toMatchSnapshot();
});
