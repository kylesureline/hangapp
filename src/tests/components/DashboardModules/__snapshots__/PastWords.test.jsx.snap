// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PastWords correctly 1`] = `
<div>
  <h3
    className="module__title"
  >
    Last 10 Words
  </h3>
  <Connect(PastWordsList) />
</div>
`;
